pytest/tests/replay/replay.py

```diff
@@ -0,0 +1,161 @@
+from account import Account
+from collections import OrderedDict
+from key import Key
+from messages.tx import *
+from messages.crypto import AccessKey, crypto_schema, PublicKey, Signature
+from messages.bridge import bridge_schema
+from serializer import BinarySerializer
+import mocknet_helpers
+
+import argparse
+import base58
+import base64
+import hashlib
+import json
+import os
+
+def generate_new_key():
+    return Key.implicit_account()
+
+def save_genesis_with_new_key_pair(genesis_path, key_pair, output_path):
+    NODE0_DIR = os.path.join(output_path, 'node0/')
+    if not os.path.exists(NODE0_DIR):
+        os.makedirs(NODE0_DIR)
+    with open(genesis_path) as fin:
+        genesis = json.load(fin)
+
+    some_validator_id = None
+    new_key = key_pair.pk.split(':')[1] if ':' in key_pair.pk else key_pair.pk
+    for validator in genesis['validators']:
```

Comment:
we probably want to remove all the validators - and leave just one. (otherwise we'd have to start multiple nodes locally).